src/nft/components/explore/CarouselCard.tsx

```diff
@@ -113,6 +108,22 @@ const CollectionImage = styled.img`
   border-radius: 100px;
 `
 
+const LoadingCardHeaderContainer = styled.div`
+  position: relative;
+  animation: ${loadingAnimation} 1.5s infinite;
+  animation-fill-mode: both;
+  background: linear-gradient(
+    to left,
+    ${({ theme }) => theme.backgroundInteractive} 25%,
+    ${({ theme }) => theme.backgroundOutline} 50%,
+    ${({ theme }) => theme.backgroundInteractive} 75%
+  );
+  will-change: background-position;
+  background-size: 400%;
+`
+const LoadingCarouselCardBorder = styled.div`
```

Comment:
tip: try https://styled-components.com/docs/api#as-polymorphic-prop instead of creating a new component